temporal-sdk/src/main/java/io/temporal/activity/ActivityCancellationType.java

```diff
@@ -19,27 +19,21 @@
 
 package io.temporal.activity;
 
-import io.temporal.failure.CanceledFailure;
-import io.temporal.workflow.CancellationScope;
-
-/**
- * Defines behaviour of the parent workflow when {@link CancellationScope} that wraps child workflow
- * execution request is canceled. The result of the cancellation independently of the type is a
- * {@link CanceledFailure} thrown from the child workflow method.
- */
+/** Defines Activity cancellation behavior. */
 public enum ActivityCancellationType {
   /**
-   * Wait for activity cancellation completion. Note that activity must heartbeat to receive a
-   * cancellation notification. This can block the cancellation for a long time if activity doesn't
-   * heartbeat or chooses to ignore the cancellation request.
+   * Wait for the Activity to complete its cancellation. Note that an Activity must Heartbeat to
```

Comment:
Something like below might be a bit more illuminating
```suggestion
   * Wait for the Activity to confirm any requested cancellation. Note that an Activity must Heartbeat to
```